scripts/bigshot.lic

```diff
@@ -4,35 +4,38 @@
  
         hunting script for hunting
  
-              author: nobody
+              author: elanthia-online
         contributers: SpiffyJr, Tillmen, Kalros, Hazado

```

Comment:
You should add yourself as a contributor especially since you are the de facto maintainer at this point.

Reply:
good point.